### PR TITLE
[12.0][NEW] Purchase Priority

### PR DESCRIPTION
**Pull request by mileo**
https://user-images.githubusercontent.com/1975978/107078785-9757c380-67cd-11eb-98da-e547aa6d6387.mp4

### CODE REVIEW

**Comment by mileo** [comment]
@gabrielcardoso21 @marcelsavegnago Could you review please?

File: purchase_priority/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 KMEE
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Purchase Priority',
+    'summary': """Priority Star on Purchase Orders""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'KMEE,Odoo Community Association (OCA)',
```
Comment (marcelsavegnago) [inline purchase_priority/__manifest__.py:9]: Add module category please.

File: purchase_priority/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[ This file must be max 2-3 paragraphs, and is required. ]
```
Comment (marcelsavegnago) [inline purchase_priority/readme/DESCRIPTION.rst:1]: Remove this line

File: purchase_priority/readme/HISTORY.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[ The change log. The goal of this file is to help readers
```
Comment (marcelsavegnago) [inline purchase_priority/readme/HISTORY.rst:1]: ?

File: purchase_priority/readme/CONFIGURE.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+[ This file is optional, it should explain how to configure
```
Comment (marcelsavegnago) [inline purchase_priority/readme/CONFIGURE.rst:1]: remove please

File: purchase_priority/readme/INSTALL.rst [hunk 5]
```diff
@@ -0,0 +1,7 @@
+[ This file must only be present if there are very specific
```
Comment (marcelsavegnago) [inline purchase_priority/readme/INSTALL.rst:1]: idem

File: purchase_priority/readme/USAGE.rst [hunk 6]
```diff
@@ -0,0 +1,14 @@
+[ This file must be present and contains the usage instructions
```
Comment (marcelsavegnago) [inline purchase_priority/readme/USAGE.rst:1]: ?

**Review by marcelsavegnago — COMMENTED** [review]
Check the readme files. At first it contains some sample data that can be removed

**Pull request closed without merging**